### PR TITLE
Fixes to readme and Configure script

### PR DESCRIPTION
**Pull request by ivernus**
Minor fix to Configure documentation and addition of CentOS7/ip compatible interface listing code.

### CODE REVIEW

File: release/README.txt [hunk 1]
```diff
@@ -5,8 +5,8 @@ Please report and bugs or feature requests by opening an issue at https://github
 Basic Moloch Installation steps:
  1) Download a Moloch build for 64bit Ubuntu 14.04, Ubuntu 16.04, Centos 6, or Centos 7 from http://molo.ch/index.html#downloads
  2) Install package
- 3) Configure basic moloch items (only need to do once)
-     /data/moloch/bin/Configure
+ 3) Configure basic moloch items by running the Configure script (this needs to be done only once)
+     . /data/moloch/bin/Configure
```
Comment (awick) [inline release/README.txt]: Why the ". "?  Configure should be executable already

**Comment by ivernus** [comment]
Hmm I'm not sure it wasnt on my system but I'll double check

File: release/Configure [hunk 2]
```diff
@@ -20,7 +20,15 @@ fi
 ### Ask config questions
 if [ -z "$MOLOCH_INTERFACE" ]; then
     echo -n "Found interfaces: "
-    /sbin/ifconfig | grep "^[a-z]" | cut -d: -f1 | cut -d" " -f1 | paste -s -d";"
+    if [ -f /etc/redhat-release ];
```
Comment (awick) [inline release/Configure]: instead of checking for redhat should it just check for ifconfig so it might work on any OS without ifconfig?

**Comment by ivernus** [comment]
Yep that makes sense @awick just made the changes now as requested

**Comment by awick** [comment]
thanks!

**Pull request merged**